client/app/components/Input/InputRadio.jsx

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React, { useState } from 'react';
+import css from './InputRadio.scss';
+import type { Option } from './utils';
+
+export type Props = {
+  name?: string,
+  value?: any,
```

Comment:
We can replace `any` with `boolean`. If `value` is undefined, we can assume it's `false`.

Reply:
In this case, InputRadio.jsx represents a collection of radio buttons so value represents which radio button is checked (tables, mg, or ml) and not whether a singe radio button is checked or not. This suggestion made me think that maybe the naming of this component is misleading since it does not represent a single radio button but instead of a collection of radio buttons were value represents which radio button is checked.

Reply:
I can change the name of this component to `InputRadioGroup.jsx`? Let me know your thoughts

Reply:
Oh sorry I read this after the previous comment. Yeah I think you should rename it! I don't think it needs to be controlled component if you wrap things in a `<fieldset>`.

Reply:
I have changed the name and made it uncontrolled on my latest commit 👍